spaceship/lib/spaceship/connect_api/testflight/testflight.rb

```diff
@@ -8,15 +8,11 @@ module TestFlight
       #
 
       def get_apps(filter: {}, includes: nil, limit: nil, sort: nil)
-        # GET
-        # https://appstoreconnect.apple.com/iris/v1/apps
```

Comment:
Just a note: I actually liked these comments as it made clear what the method is doing, without having to look at the actual code. Maybe worth keeping as a comment for the method (so above it)?

Reply:
Maybe also with a link to the API documentation? (if it has a proper page per endpoint that one could link to here...)

Reply:
The weird thing is those end points actually change if you are using web session or JWT token so its not 100% accurate 😔 I think a link to the docs might be kinda nice instead 🤔